test/rollbax/client_test.exs

```diff
@@ -23,8 +23,7 @@ defmodule Rollbax.ClientTest do
     assert body =~ ~s("environment":"test")
     assert body =~ ~s("level":"warn")
     assert body =~ ~s("body":"pass")
-    assert body =~ ~s("foo":"bar")
-    assert body =~ ~s("qux":"custom")
+    assert Poison.decode!(body)["data"]["custom"] == %{"foo" => "bar", "qux" => "custom"}
```

Comment:
@SimonWoolf I meant doing that in the whole test suite actually, so I would prefer to do that on a separate PR so that this PR remains very focused and tiny :) Would that be a problem for you?